src/components/Popover/LazyTippy.tsx

```diff
@@ -0,0 +1,36 @@
+/** @jsxImportSource theme-ui */
+import { forwardRef, useState } from "react"
+import Tippy, { TippyProps } from "@tippyjs/react"
+
+const LazyTippy = forwardRef<HTMLElement, TippyProps>(function LazyTippy(
+  props: TippyProps,
+  ref
+) {
+  const [mounted, setMounted] = useState(false)
+
+  const lazyPlugin = {
+    fn: () => ({
+      onMount: () => setMounted(true),
+      onHidden: () => setMounted(false),
+    }),
+  }
+
+  const computedProps = { ...props }
+
+  computedProps.plugins = [lazyPlugin, ...(props.plugins || [])]
+
+  if (props.render) {
+    computedProps.render = (...arguments_) =>
+      mounted && props.render ? props.render(...arguments_) : ""
```

Comment:
Is this check needed: `mounted && props.render`?
This is already inside an if that checks if `props.render` is truthy, so it should be the same as just `mounted`

Reply:
TS does not allow me to invoke a method that can be possibly undefined, so I'm checking right before, I didn't find a best solution for it, so just jumped to the most usual solution. kinda ugly indeed.